system_test/common/aest_db_SUITE.erl

```diff
@@ -53,8 +55,12 @@ node_can_reuse_db_of_other_node(Cfg) ->
 roma_node_can_reuse_db_of_other_roma_node(Cfg) ->
     node_can_reuse_db_of_other_node_(fun roma_node_spec/2, Cfg).
 
-node_can_reuse_db_of_roma_node(Cfg) ->
-    node_can_reuse_db_of_other_node_(fun roma_node_spec/2, fun node_spec/2, Cfg).
+minerva_node_with_epoch_db_can_reuse_db_of_roma_node(Cfg) ->
```

Comment:
This is for 2.0.0* to be able to start from 1.4.0 DB - because same name in DB. Ok.